lib/misc.c

```diff
@@ -1282,3 +1282,38 @@ gftp_get_share_dir (void)
   return (gftp_share_dir);
 }
 
+char *
+gftp_format_file_size(off_t bytes) {
+  char tmp[20];
+  char ret[40];
+  char *pointzero = 0;
+  off_t TB = 1024LL * 1024LL * 1024LL * 1024LL;
+  off_t GB = 1024LL * 1024LL * 1024LL;
+  off_t MB = 1024LL * 1024LL;
+  if (bytes >= TB) {
+    snprintf(tmp, sizeof(tmp), "%.2f", (double) bytes / TB);
+    pointzero = strstr(tmp, ".00");
```

Comment:
Very minor nitpick: This should be ".0" instead of ".00" to match the others for consistency. I have no preference between .0 and .00, just as long as we go with one.